### DISCUSSION TITLE
Help on estimated plot size error on spacepool

### DISCUSSION THREAD

**Discussion by AbleT4908** [Chia-Network/chia-blockchain#8997] [Support]
![화면 캡처 2021-10-28 100222](https://user-images.githubusercontent.com/93184655/139168596-d428f881-6553-4941-b182-e5cf634a2ef0.png)

i suppose to have over 2PiB, but on the pool it only shows 1.425 Pib. The wierd thing is, it moves up and down but not to the maximum amount of plot size I've got.
Sometime it became 1.75 PiB, and the other day it shows 1.2 PiB. Does anyone know the fix of this? Or it doesn't really matter, because its just a show on a page?

**Comment by FFurno** [comment]
It has happened to me as well. Try to check the following:
- all the hard drives and their pools are properly recognized 
- you have enough cpu power available
- your internet connection is working well

**Reply by FFurno** [reply]
their plots**

**Reply by mazm786** [reply]
How **estimated plot size** (for pool) is calculated or read from pool or from Chia software 
?

**Comment by mazm786** [comment]
How estimated plot size (for pool) is calculated or read from pool or from Chia software
?

**Reply by loppefaaret** [reply]
i guess spacepool is estimating your total plot size based on the partials that you submit to their pool - if you have problems submitting partials, your total size can suffer
